src/SDKs/Azure.Core/data-plane/Azure.Core/ResponseHeaders.cs

```diff
@@ -16,6 +17,12 @@ public ResponseHeaders(Response response)
             _response = response;
         }
 
+        public DateTimeOffset? Date => TryGetValue(HttpHeader.Names.Date, out var value) ? (DateTimeOffset?)DateTimeOffset.Parse(value) : null;
```

Comment:
>public DateTimeOffset? Date => TryGetValue(HttpHeader.Names.Date, out var value) ? (DateTimeOffset?)DateTimeOffset.Parse(value) : null; [](start = 8, length = 135)

Should we be throwing an exception in the case that the date format is not supporeted?

Reply:
My thought was to throw and see if it would be a problem for anyone.